src/platforms/windows/windowsnetworkwatcher.cpp

```diff
@@ -40,7 +40,7 @@ void WindowsNetworkWatcher::initialize() {
 
   DWORD prefNotifSource;
   if (WlanRegisterNotification(m_wlanHandle, WLAN_NOTIFICATION_SOURCE_MSM,
-                               true /* ignore duplicates */, wlanCallback, this,
+                               true /* ignore duplicates */, (WLAN_NOTIFICATION_CALLBACK) wlanCallback, this,
```

Comment:
This compile error popped out of nowhere. 
msvc was not able to cast wlanCallback automatically?